### PR TITLE
refactor: Change variable name FraudProofs to StateFraudProofs

### LINKED ISSUE #637: Change variable name FraudProofs to StateFraudProofs
The variable name should be more descriptive as we are working with a lot of different FraudProofs in the Celestia Ecosystem. 
The change in name should encompass all mentions of FraudProofs in Rollmint, Testcases and Log messages.

### PR DESCRIPTION
**Pull request by chandiniv1**
closes #637

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -53,9 +53,9 @@ type BlockManagerConfig struct {
 	// DABlockTime informs about block time of underlying data availability layer
 	DABlockTime time.Duration `mapstructure:"da_block_time"`
 	// DAStartHeight allows skipping first DAStartHeight-1 blocks when querying for blocks.
-	DAStartHeight uint64            `mapstructure:"da_start_height"`
-	NamespaceID   types.NamespaceID `mapstructure:"namespace_id"`
-	FraudProofs   bool              `mapstructure:"fraud_proofs"`
+	DAStartHeight    uint64            `mapstructure:"da_start_height"`
+	NamespaceID      types.NamespaceID `mapstructure:"namespace_id"`
+	StateFraudProofs bool              `mapstructure:"fraud_proofs"`
```
Comment (Manav-Aggarwal) [inline config/config.go:58]: changing ``mapstructure:"fraud_proofs"`` to ``mapstructure:"state_fraud_proofs"`` would make sense as well.

**Review by Manav-Aggarwal — CHANGES_REQUESTED** [review]
Left some comments. Rest looks good, thanks!

**Review by Manav-Aggarwal — APPROVED** [review]
LGTM

**Review by nashqueue — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**